### ISSUE TITLE
Extra Data Randomly Appended to Output Files

### ISSUE THREAD

**Issue by AlexRamey** [bentonam/fakeit#153]
The output files sometimes will randomly have extra data appended after the final closing brace of the generated model. This seems like some buffer flushing issue. Not sure what's causing it. The random data doesn't seem to be coming from other files in the same run or from files generated by the previous run, but it's possible I missed something.

Here is some actual output. Everything after the closing brace (starting with "lastDayStatus") should not be there. I've added ** around what should be gone.
```
. . .
  "Created": 1503414324809,
  "createdBy": "esb-user",
  "Modified": 0,
  "modifiedBy": ""
}   **"lastDayStatus": "LOWSTOCK",
        "lastDayValue": 86
      },
      "newProductInfo": {
        "startDate": "2017-8-15",
        "endDate": "2017-8-22"
      },
      "merchandizeLayout": {
        "module": "target",
        "moduleDescription": "Kentucky",
        "area": "e-tailers",
        "aisle": "Forward",
        "sortSequence": "Maryland"
      },
      "listingType": "Assortment",
      "lockInfo": {
        "severity": "Fords",
        "description": "Tuna",
        "validFrom": "2017-8-16"
      }
    }
  ],
  "Created": 1503414324766,
  "createdBy": "esb-user",
  "Modified": 0,
  "modifiedBy": ""
}**
```

**Comment by AlexRamey** [comment]
UPDATE: I think the problem is that I'm generating multiple documents with the same file name in a given run sometimes. So I think the second one overwrites the first one, but if the first one was longer then its tail end is still there.

**Comment by AlexRamey** [comment]
solved by PR #170

**Comment by AlexRamey** [comment]
solved by PR #170
